tests/specs/erc20/verification.k

```diff
@@ -25,6 +25,15 @@ module VERIFICATION-JAVA [symbolic, kast]
 endmodule
 
 module VERIFICATION-HASKELL [symbolic, kore]
-    import INFINITE-GAS
+    imports INFINITE-GAS
+    imports EDSL
+
+    rule #hashedLocation(_COMPILER, BASE, .IntList) => BASE [simplification]
```

Comment:
You shouldn't need this lemma, it's duplicate of a lemma here: https://github.com/kframework/evm-semantics/blob/master/hashed-locations.md#hashed-location-for-storage

Reply:
All of them are duplicates of lemmas there. I added them incrementally and they each made progress, but I didn't try different combinations of them. I'll try removing this one and see if it still passes.